docs/troubleshooting.md

```diff
@@ -39,15 +39,8 @@ and you can ask for new features [in their repository](https://github.com/Micros
   This is expected until VSCode offers extension channels - [vscode#15756](https://github.com/Microsoft/vscode/issues/15756)
   - You MUST [DISABLE](https://code.visualstudio.com/docs/editor/extension-gallery#_disable-an-extension) one of them for the best performance.
     Docs on how to disable an extension can be found [here](https://code.visualstudio.com/docs/editor/extension-gallery#_disable-an-extension)
-- Highlighting/completions/command history don't work as I expect in the
-  Integrated Console - [#535]
-  - The Integrated Console implements a [custom host]
-    to work with VSCode, meaning that functionality could be different than that of the regular host in the PowerShell Console
-  - [PSReadLine] (the module providing these features in regular PowerShell) is available in the PowerShell Preview Extension, helping to bridge this gap
-  - Making PSReadline fully available is being actively worked on.
-- Command history is not preserved when debugging in the Integrated Console -
-  [#550]
-  - This feature is also provided by [PSReadLine].
+- The extension won't start but it does start with the x86 version of PowerShell
+  - Do you use Avecto/BeyondSoft? We've received reports that Avecto, BeyondSoft and other privilege management software dramatically slow down the start up of Windows PowerShell x64. Please give the privilege management software feedback. For some, updating to the latest version has fixed the issue.
```

Comment:
May also be worth linking to an existing issue here so we can try to collect all the Avecto people in one place

```suggestion
  - Do you use Avecto/BeyondSoft?
    We've received reports that Avecto, BeyondSoft
    and other privilege management software
    dramatically slow down the start up of Windows PowerShell x64.
    Please give the privilege management software feedback.
    For some, updating to the latest version has fixed the issue.
```